docs/build-html.script

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
would you mind changing the extension to something more descriptive like `.javascript`, maybe? `script` could be literally anything.

Reply:
applied suggestion in commit 218e696e